README.md

```diff
@@ -1,22 +1,22 @@
-# skyux-cli
+# @skyux-sdk/cli
 
 [![npm](https://img.shields.io/npm/v/@blackbaud/skyux-cli.svg)](https://www.npmjs.com/package/@blackbaud/skyux-cli)
```

Comment:
Doesn't have to be this PR, but we should update these to point to the new `npm` package.

Reply:
Done!